storage/innobase/include/ut0ut.h

```diff
@@ -88,15 +88,25 @@ struct ut_when_dtor {
    the YieldProcessor macro defined in WinNT.h. It is a CPU architecture-
    independent way by using YieldProcessor. */
 #  define UT_RELAX_CPU() YieldProcessor()
-# elif defined(HAVE_ATOMIC_BUILTINS)
```

Comment:
Atomic operations aren't relaxing on any CPU and have effects beyond the CPU which is counter to what ut_delay wants to achieve.

@akopytov do you have a suggestion as to what ARM should do to relax a processor?
@akamat-ibm any suggestion on what Z should do?